logs/plugin_bot.go

```diff
@@ -30,6 +31,12 @@ func (p *Plugin) AddCommands() {
 		commands.AddRootCommands(p, cmdLogs, cmdWhois, cmdNicknames, cmdUsernames, cmdClearNames)
 	} else {
 		commands.AddRootCommands(p, cmdLogs, cmdWhois)
+		if run.FlagGenCmdDocs {
+			for _, cmd := range []*commands.YAGCommand{cmdNicknames, cmdUsernames, cmdClearNames} {
+				cmd.Description = fmt.Sprint(cmd.Description, " Disabled on the public instance.")
```

Comment:
Perhaps "disabled on this instance" instead? Though I'm not sure if the average self-hoster bothers to host their own docs, let alone generate command docs, so maybe not.

Reply:
That's my thought as well, if you're using gencmddocs you can always mod the source code.